### PR TITLE
Validation form

### PR DESCRIPTION
**Pull request by LYANGEND**
**Done the Following Activities**:
For this milestone of my portfolio website, I have implemented a client-side validation of the data in the contact form before submission. When the user Enters Uppercase letters in the email tab then the form will not be sent. The Email has to be Lower case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ to run the project locally, clone the project using the command
 navigate in to the project directory, from GitHub,
 to test or run the application double click on the `index.html` file to open in your browser
 
-## Authors
+## Author
 
 👤 **Author**
```
Comment (elmar8287) [inline README.md:41]: - [ ] Please, consider fixing the broken screenshot in your README file 👍 
Screen Shot 2022-03-04 at 22 35 29

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @LYANGEND.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

To highlight:
- Nice job with the implementation of validation 👍 
- No linter issues 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.
- [ ] You did create a job with the implementation of validation for the contact form, but you have to place the error message near the submit button, as it is required in the project. 
- [ ] Please, additionally, I suggest to you to style the error message more visible, aka give the background to make it more clearly visible. In your version, it is hard to read the error message 👍 
Screen Shot 2022-03-04 at 22 42 29



## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LYANGEND** [comment]
hey @elmar8287, thank you for the effort done to review this code. But don't think its right for the error to be beneath the email section to help the user realize where the mistake is ?? I feel if I put it under the message section it would not be effective for the email section. with reference to a lot of sites that have been reviewed, I think this is the right position.

please share your thought.

Regards David.

**Comment by elmar8287** [comment]
> hey @elmar8287, thank you for the effort done to review this code. But don't think its right for the error to be beneath the email section to help the user realize where the mistake is ?? I feel if I put it under the message section it would not be effective for the email section. with reference to a lot of sites that have been reviewed, I think this is the right position.
> 
> please share your thought.
> 
> Regards David.

Dear @LYANGEND. First of all, thank you for your collaboration. Yes, I can agree with your opinion, but I gave this suggestion because of the project requirements. It says, to put the error message near the submit button. So, I know that you can do it easily, and also, as I mentioned in the review, try to give a background for the error message to make it more readable on the contact form background. I am at your disposal if you have any questions or support from my side. Happy coding! 🥇

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @LYANGEND 👋🏻, 

Good job on implementing the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**